irispy/sji.py

```diff
@@ -878,3 +879,73 @@ def dustbuster(mc):
         map.data = image_orig
 
     return mc
+
+
+def read_iris_sji_level2_fits(filename):
+    """
+    Read IRIS level 2 SJI FITS from an OBS into an NDCube instance
+
+    Parameters
+    ----------
+    filename : `str`
+        File name to be read
+
+    Returns
+    -------
+    result: 'ndcube.NDCube'
+
+    """
+
+    # Open a fits file
+    my_file = pyfits.open(filename)
```

Comment:
I don't know if this was your choice or not but I am *strong* :-1: on `pyfits`, the recommended way from astropy and sunpy is:

```python
from astropy.io import fits

fits.open(....
```

Reply:
I know but on the sji.py file, pyfits is used : this is not my choice and I don't want to change this part. However, for my next pull request, I can modify it if you want (I also prefer fits than pyfits !)

Reply:
If you want to change it in a separate PR that sounds awesome!

Reply:
In fact, ```astropy.io.fits``` is already being used.  See the import statement which says:
```from astropy.io import fits as pyfits``` (https://github.com/sunpy/irispy/pull/67/files#diff-c3a5b1ac9cb9013bd70b2db039f006b3R15)
@BaptistePellorceAstro, could you delete the  ```as pyfits``` part of that line and then to a search and replace changing ```pyfits``` to ```fits```?

Reply:
Yes, I planned to do it.

Reply:
👍